src/main/kotlin/no/nav/syfo/metric/Metrics.kt

```diff
@@ -56,5 +56,12 @@ const val PERSONOPPGAVEHENDELSE_DIALOGMELDING_SVAR_MOTTAT =
     "${METRICS_NS}_personoppgavehendelse_dialogmelding_svar_mottat_count"
 val COUNT_PERSONOPPGAVEHENDELSE_DIALOGMELDING_SVAR_MOTTAT: Counter =
     Counter.builder(PERSONOPPGAVEHENDELSE_DIALOGMELDING_SVAR_MOTTAT)
-        .description("Counts the number of personoppgavehendelse with PersonoppgavehendelseType DIALOGMELDING_MOTTAT created from a KMeldingFraBehandler")
+        .description("Counts the number of personoppgavehendelse with PersonoppgavehendelseType BEHANDLERDIALOG_SVAR_MOTTATT created from a KMeldingFraBehandler")
+        .register(METRICS_REGISTRY)
+
+const val PERSONOPPGAVEHENDELSE_UBESVART_MELDING_MOTTAT =
```

Comment:
Kanskje man skulle kalt disse `PERSONOPPGAVEHENDELSE_DIALOGMELDING_UBESVART_MOTTATT` i stedet? Blir mer konsistent med hendelsen over og høres mer ut som at det er en hendelse (DIALOGMELDING_UBESVART) og ikke en ubesvart melding som er mottatt.
Og så er det vel mottatt med to t-er til slutt? ✏️

Reply:
Jo, vi kan godt det! Og ja, det er med to t-er, jeg har bare blingsa 🤷🏼‍♂️

Reply:
Kanskje jeg faktisk skal få lagt inn metricene i grafana også 😬 Det er jo ikke dumt